src/sentry/uptime/grouptype.py

```diff
@@ -131,19 +131,12 @@ def build_event_data(result: CheckResult, detector: Detector) -> EventData:
     # Received time is the actual time the check was performed.
     received = datetime.fromtimestamp(result["actual_check_time_ms"] / 1000)
 
-    # XXX(epurkhiser): This can be changed over to using the detector ID in the
-    # future once we're no longer using the ProjectUptimeSubscription.id as a tag.
-    project_subscription = get_project_subscription(detector)
```

Comment:
There's a way to query for detector's in issues using search now. So we don't even need to store it as a tag.